e2e/testcafe-devextreme/tests/accessibility/multiView.ts

```diff
@@ -0,0 +1,32 @@
+import { Properties } from 'devextreme/ui/multi_view.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const items = ['Item_1', 'Item_2', 'Item_3'];
+
+const options: Options<Properties> = {
+  dataSource: [[], items],
+  height: [300],
+  loop: [true, false],
+  hint: [undefined, 'hint'],
+  noDataText: [undefined, 'no data text'],
+  swipeEnabled: [true, false],
```

Comment:
Does this option add any new elements to the component? Or does it change attributes?